test/integration/generated/iotspaces/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "iotspaces",
+  "sdk-type": "management",
+  "author": "Microsoft Corporation",
+  "description": "A generated SDK for IoTSpacesClient.",
+  "version": "1.0.0-preview1",
+  "engines": { "node": ">=12.0.0" },
+  "dependencies": {
+    "@azure/core-client": "^1.1.2",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-rest-pipeline": "1.0.0-beta.2",
```

Comment:
why are generating a beta version here?

Reply:
I think this is a left over. Changed it now